README.md

```diff
@@ -1,5 +1,5 @@
-[![Build Status](https://travis-ci.org/smartdevicelink/sdl_java_suite.svg?branch=master)](https://travis-ci.org/smartdevicelink/sdl_java_suite)
-[![codecov](https://codecov.io/gh/smartdevicelink/sdl_android/branch/master/graph/badge.svg)](https://codecov.io/gh/smartdevicelink/sdl_java_suite)
+[![Build Status](https://github.com/smartdevicelink/sdl_java_suite/workflows/GitHub%20CI/badge.svg?branch=master)](https://github.com/smartdevicelink/sdl_java_suite/actions)
```

Comment:
When I view this file, this badge isn't working correctly

Reply:
You are right. But that is just because `master` branch doesn't have the Git Actions setup. So ths master one currently doesn't work: 
https://github.com/smartdevicelink/sdl_java_suite/workflows/GitHub%20CI/badge.svg?branch=master

However, the develop one works
https://github.com/smartdevicelink/sdl_java_suite/workflows/GitHub%20CI/badge.svg?branch=develop

Also the current branch one:
https://github.com/smartdevicelink/sdl_java_suite/workflows/GitHub%20CI/badge.svg?branch=feature/github_actions_codecov

Reply:
Gotcha